R/setup_project.R

```diff
@@ -0,0 +1,73 @@
+setup_project <- function(path, ...) {
+  # collect inputs
+  dots <- list(...)
+
+  # create directory
+  dir.create(path, recursive = TRUE, showWarnings = FALSE)
+  # create img folder within  path
+  dir.create(paste0(path, "/img"), recursive = TRUE, showWarnings = FALSE)
+  # create img/box-images folder within path
+  dir.create(paste0(path, "/img/box-images"), recursive = TRUE, showWarnings = FALSE)
+  # create .github folder within path
+  dir.create(paste0(path, "/.github"), recursive = TRUE, showWarnings = FALSE)
+  # create .github/workflows folder within path
+  dir.create(paste0(path, "/.github/workflows"), recursive = TRUE, showWarnings = FALSE)
+
+  # Move boilerplate files into path
+
+  # Function to copy each file
+  copy_files <- function(file_name, style_set) {
```

Comment:
Might be good to break out this function so it is its own function instead of defining it within another function.

Reply:
Good idea. I've incorporated this in https://github.com/jhudsl/ottrpal/pull/128/commits/9c1479c50c9694f1c9f74f1c9a51cb9f4e41c6ce